### PR TITLE
Make CI more similar to (failing) nightly builds.

### PR DESCRIPTION
**Pull request by arosien**
Nightly uses `vscode: [ 'stable', 'insiders' ]`, whereas CI doesn't specify the vscode version, which defaults to the `package.json` vscode engine version, currently set to `^1.67.2`, which gets resolved to `1.67.2`.

However, let's make the builds as similar as possible to provoke the same failures in CI as nightly.

This provokes the same failures as nightly, and the fix is to not attempt to delete the extracted backend debugger directory. This was added to make local development safer, but re-running the tests locally will always delete the extracted backend debugger before re-extracting it, which happens during the webpack step (using webpack/ext-dev.webpack.config.js).

### CODE REVIEW

**Comment by arosien** [comment]
Yay I got the same failures as nightly, now I can try to find a proper fix.

**Review by NolanMatt — APPROVED** [review]

**Review by shanedell — APPROVED** [review]
+1 👍

**Comment by shanedell** [comment]
@arosien For future reference you need to have 2 green approvals before merging.

**Comment by arosien** [comment]
> @arosien For future reference you need to have 2 green approvals before merging.

ah, **green** is the important word here. got it! (there were failed tests due to network timeouts...)

**Pull request merged**
